api/tripal_galaxy.api.inc

```diff
@@ -321,12 +321,18 @@ function tripal_galaxy_get_history_name($submission, $node) {
  *  @param $uid
  *    The user id of the person who submitted the workflow.
  */
-function tripal_galaxy_invoke_workflow($galaxy, $workflow_id, $parameters, 
+function tripal_galaxy_invoke_workflow($galaxy, $workflow_id, $parameters,
```

Comment:
Here's an example of a change that I think will break things.  We have changed the function `function tripal_galaxy_invoke_workflow` to look like this:

https://github.com/tripal/tripal_galaxy/blob/7a4bb4bc4fd4a7c45954d872e0babb8c7cbf324a/api/tripal_galaxy.api.inc#L324-L329

But in this PR it alters that. I think it's trying to reset it to what it was before: